code/game/gamemodes/cult/structures/pylon.dm

```diff
@@ -329,11 +343,32 @@
 	A = null //So projectiles can GC
 	addtimer(CALLBACK(src, PROC_REF(handle_firing)), shot_delay + 1)
 
-/obj/structure/cult/pylon/attack_hand(mob/M)
-	if (M.a_intent == "help")
-		to_chat(M, SPAN_WARNING("The pylon feels warm to the touch..."))
+/obj/structure/cult/pylon/attack_hand(mob/living/user)
+	if (user.a_intent == I_HELP)
+		to_chat(user, SPAN_WARNING("The pylon feels warm to the touch..."))
+
+	else if((user.a_intent == I_DISARM) && (pylonmode != PYLON_TURRET))
```

Comment:
Disarm seems like an odd intent, this should probably be made obvious somewhere because otherwise it's rather unintuitive

Reply:
Added in the antag description text how to do this